### PR TITLE
add kubesphere to the partner

### PR DESCRIPTION
**Pull request by faweizhao26**

### CODE REVIEW

File: src/pages/partners/partners.mdx [hunk 1]
```diff
@@ -12,4 +12,4 @@ hide_table_of_contents: true
...
 | Huawei Cloud           |   IEF    | Edge computing platform built upon KubeEdge, provides extremely lightweight, edge intelligent, and powerful in terms of computing capabilities.        | https://www.huaweicloud.com/product/ief.html     |
-
+| KubeSphere |   KubeSphere Enterprise 4.0   | KubeSphere Enterprise (KSE) 4.0 is a new cloud-native operating system built by QingCloud Technologies, which has extraordinary epoch-making significance for KubeSphere. Based on the open and extensible architecture of KubeSphere LuBan, KubeSphere Enterprise 4.0 can easily achieve upstream and downstream linkage of applications, integrate various high-quality extensions from KubeSphere Marketplace anytime, and provide seamless business capabilities and highly consistent product experience.     | https://kubesphere.co/kse/   |
```
Comment (fisherxu) [inline src/pages/partners/partners.mdx:15]: Could you plz add some description related to edge computing capabilities in the description section? Thanks :) @faweizhao26

Comment (faweizhao26) [reply src/pages/partners/partners.mdx]: > Could you plz add some description related to edge computing capabilities in the description section? Thanks :) @faweizhao26

Done.

Comment (fisherxu) [reply src/pages/partners/partners.mdx]: Okey, thanks!

**Review by fisherxu — COMMENTED** [review]
/approve
/assign @kubeedge/tsc @kubeedge/maintainers

**Pull request merged**